PWGDQ/Tasks/dqCorrelation.cxx

```diff
@@ -0,0 +1,512 @@
+
+  /////////////////////
+
+  Configurable<float> fConfigMuonCutEtaMin{"cfgCutEtaMinMuon", -4.f, "Eta min range for muons"};
+  Configurable<float> fConfigMuonCutEtaMax{"cfgCutEtaMaxMuon", -2.5f, "Eta max range for muons"};
+
+  Configurable<float> fConfigMultiplicityMin{"cfgMultiplicityMin", 3500.0f, "Multiplicity for 30%"};
+  Configurable<float> fConfigMultiplicityMax{"cfgMultiplicityMax", 2000.0f, "Multiplicity for 60%"};
+
+  /////////////////////
+
+  // Configurables for the dilepton and dilepton cuts
+  Configurable<float> fConfigDileptonLowMass{"cfgDileptonLowMass", 2., "Low mass cut for the dileptons used in analysis"};
+  Configurable<float> fConfigDileptonHighMass{"cfgDileptonHighMass", 4., "High mass cut for the dileptons used in analysis"};
+
+  // Connect to ccdb
+  Service<ccdb::BasicCCDBManager> ccdb;
+  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
+  Configurable<std::string> url{"ccdb-url", "http://ccdb-test.cern.ch:8080", "url of the ccdb repository"};
+
+  // // // CCDB access
+  // Service<ccdb::BasicCCDBManager> ccdb;
+  // // CCDB access for the efficiencies and acceptances
+  // Configurable<std::string> fConfigEfficiency{"ccdb-path-efficiency", "Users/r/rcaron/efficiency", "CCDB path to efficiency object"};
+  // Configurable<std::string> fConfigAcceptance{"ccdb-path-acceptance", "", "CCDB path to acceptance or GFWWeights object"};
+  // Configurable<std::string> fConfigURL{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
+  // Configurable<int64_t> fConfigNoLaterThan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
+
+  // Configurables for histograms
+  Configurable<std::string> fConfigAddDileptonHadHistogram{"cfgAddDileptonHadHistogram", "", "Comma separated list of histograms"};
+
+  // Configurables for FlowContainer (e.g charged particles pt-differential v2{2}, v2{3}, ...)
+  ConfigurableAxis axisMultiplicity{"axisMultiplicity", {VARIABLE_WIDTH, 0, 5, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100.1}, "multiplicity / centrality axis for histograms"};
+
+  ////////////////////////////////////////////////////////////////////////////////
+  // Define the filter for events
+  Filter eventFilter = aod::dqanalysisflags::isEventSelected == 1;
+
+  // Define the filter for barrel tracks and forward tracks
+  Filter trackFilter = (nabs(aod::track::eta) <= fConfigCutEtaMax) && (aod::track::pt > fConfigCutPtMin) && (aod::track::pt < fConfigCutPtMax);
+  Filter fwdFilter = (nabs(aod::fwdtrack::eta) <= fConfigMuonCutEtaMax) && (nabs(aod::fwdtrack::eta) >= fConfigMuonCutEtaMin);
+  // Filter fwdFilter = (aod::fwdtrack::eta < -2.f) && (aod::fwdtrack::eta > -4.5f);
```

Comment:
remove, also see below. Please remove all the lines that are not needed. This helps with readability of the diffs for future PRs